__tests__/user/user.mutation.test.js

```diff
@@ -33,10 +33,18 @@ describe('mutations', () => {
 
         userId = res.data.registerUser._id
         
+        expect(res.data.registerUser).toHaveProperty(
+            'firstName', 'Petro'
+        );
+        expect(res.data.registerUser).toHaveProperty(
+            'lastName', 'Tatsenyak'
+        );
         expect(res.data.registerUser).toHaveProperty(
             'email', 'tacjka34@gmail.com'
         );
-
+        expect(res.data.registerUser).toHaveProperty('credentials');  
```

Comment:
I don't think that credentials should be returned back to the client. It should not be part of the graphql model of user

Reply:
We can send a password but we cannot send it back

Reply:
fixed